src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="console.exception_listener" class="Symfony\Component\Console\EventListener\ExceptionListener">
```

Comment:
Since it is an event listener, it needs #21312 to be merged first (see https://travis-ci.org/symfony/symfony/jobs/192665057#L2971)